### PR TITLE
Fix absorbed Ignite damage

### PR DESCRIPTION
**Pull request by GuybrushGit**
Alternative to https://github.com/lh-server/core/pull/65/

### CODE REVIEW

File: src/game/Spells/Spell.cpp [hunk 1]
```diff
@@ -1284,6 +1284,7 @@ void Spell::DoAllEffectOnTarget(TargetInfo *target)
         }
 
         unitTarget->CalculateAbsorbResistBlock(caster, &damageInfo, m_spellInfo, BASE_ATTACK, this);
+        m_absorbed = damageInfo.absorb;
```
Comment (thepparker) [inline src/game/Spells/Spell.cpp]: Initialize this in the constructor

**Comment by thepparker** [comment]
Much better :D

File: src/game/Spells/Spell.cpp (hunk 1 above)
Comment (bladezLH) [inline src/game/Spells/Spell.cpp:1288]: Initialize this in the constructor

**Comment by bladezLH** [comment]
This is much cleaner huh :P

File: src/game/Spells/Spell.cpp (hunk 1 above)
Comment (GuybrushGit) [reply src/game/Spells/Spell.cpp:1288]: Done

**Pull request merged**